couchbase-lite-core/check_builds/pkg_data.yaml.j2

```diff
@@ -0,0 +1,66 @@
+{# Map release names to the set of expected platforms. Each entry has the form #}
+{#     platform [ : architecture ] #}
+{# If :architecture is empty, the filename will use the default arch for the #}
+{# platform's packaging style (see below). #}
+{##}
```

Comment:
Should remove these comments about :architecture - doesn't seem to correspond to the code.